package.json

```diff
@@ -6,6 +6,7 @@
     "@azure/msal-browser": "2.12.0",
     "@babel/core": "7.12.13",
     "@babel/eslint-parser": "7.12.13",
+    "@fluentui/react": "^8.22.2",
```

Comment:
This can be left out because there is no need to call it yet as the code base will be migrated to fluent UI soon

Reply:
```suggestion
```

Reply:
```suggestion
```